src/libs/ajax/User.ts

```diff
@@ -56,7 +56,7 @@ export type TerraUserPreferences = {
 
 export interface OrchestrationUserProfileResponse {
   userId: string;
-  keyValuePairs: TerraUserProfile;
+  keyValuePairs: any; // This is not a TerraUserProfile until Utils.kvArrayToObject is called on this
```

Comment:
Is there a more specific type we can use here?

```suggestion
  keyValuePairs: { key: string; value: string; }[]; // Convert to a TerraUserProfile using Utils.kvArrayToObject
```

Reply:
I would like to just have profile actually return a TerraUserProfile, so we can do the keyValuePair conversion in that method. But that touches workspaces code. Is that appropriate in this situation?

Reply:
That's fine. But either way, there's a more specific type than `any` that can be used here.

Reply:
Yep, fixing that for sure 👍